docs/src/building.md

```diff
@@ -344,3 +344,40 @@ Examples of builders that depend on other binaries include:
 * [`Xorg_libX11`](https://github.com/JuliaPackaging/Yggdrasil/blob/eb3728a2303c98519338fe0be370ef299b807e19/X/Xorg_libX11/build_tarballs.jl#L36-L42)
   depends on `Xorg_libxcb_jll`, and `Xorg_xtrans_jll` at build- and run-time,
   and on `Xorg_xorgproto_jll` and `Xorg_util_macros_jll` only at build-time.
+
+# Building and testing JLL packages locally
+
+As a package developer, you may want to test JLL packages locally, or as a binary dependency
+developer you may want to easily use custom binaries.  In both cases, the new `dev` and
+`overrides` directory support make it easy to get complete control over the local JLL package
+state.
```

Comment:
The way this is phrased (esp. the use of "the new ... support") makes it well suited for a release announcement, but IMHO somewhat less so for a permanent entry in the manual. I don't claim to know the perfect way to do this, but in an attempt to be constructive and not just destructive, here is an attempt for an alternative phrasing
```suggestion
developer you may want to easily use custom binaries. To support these use cases, we provide
means to make it easy to get complete control over the local JLL package state, which are
described in the following subsections.
```